src/workerd/api/wpt/url-test.ts

```diff
@@ -36,7 +35,7 @@ export default {
     ],
   },
   'percent-encoding.window.js': {
-    comment: 'Implement `async_test`',
+    comment: 'Implement document',
```

Comment:
We don't need to implement document but simply need the functionality to mutate the code before running them:

Example: https://github.com/nodejs/node/blob/main/test/wpt/test-url.js#L9